config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "modules": ["notebooks"]
```

Comment:
There is a requirement that customer should be able to turn on and turn off every module. Not all customers are using all features of data.all. Another case is when you want to replace some functionality with your own e.g. you don't want to use existing mechanism for sharing datasets. In this case, you just delete a line in the config and put your module name with your code.

By default, all refactored modules will be added to config.json, but then you can change it for your specific needs.

The customization can be go beyond that and instead of having just name of module we can have a configuration for every module:
```
{
    "modules" : [
           "notebooks" : {
                    "use_free_tier_only:": true,
                     "notebook_prefix": "my_company.com_notebook"
            },
           "datasets": {
                   "some_prop": false
           },
          etc
    ]
}
```

Reply:
I think we may need to distinguish between disabling the feature vs disabling the module? If someone will disable a feature in the data.all UI (notebooksEnabled set to false in the db) it would mean disabling the module?

Reply:
When we disable the module, we won't see`notebooksEnabled` in UI at all (not now, but it will be when we modularize frontend). But if the notebook module is enabled we still have a choice to disable it per environment. 
Basically, the rule is: 
if all your environments don't need notebooks/ you are not going to use it -> the notebook module should be disabled
If you are not sure or some/all your environments are planning to work with it -> the module should be enabled, but you will still have a choice to disable it for each env you want by `notebooksEnabled=false`